libs/sdk-ui-dashboard/src/presentation/widget/widget/InsightWidget/DefaultDashboardInsightWidget.tsx

```diff
@@ -143,13 +186,17 @@ const DefaultDashboardInsightWidgetCore: React.FC<
             timeout = setTimeout(() => {
                 setIgnoreHighlight(true);
             }, 5000);
+
+            if (elementRef.current && shouldScrollTo) {
+                elementRef.current.scrollIntoView({ behavior: "smooth", block: "center" });
+            }
         }
         return () => {
             if (timeout) {
                 clearTimeout(timeout);
             }
         };
-    }, [isHighlighted]);
+    }, [isHighlighted, shouldScrollTo]);
```

Comment:
should we add dependency on the automationId/visualizationId/widgetId etc somwhow? So it's reflecting, if the parameter has changed?

Reply:
Good point, I will add the properties there and probably even move the shouldScroll logic when the properties are available in the component itself